package.json

```diff
@@ -163,5 +164,8 @@
         ]
       }
     }
+  },
+  "dependencies": {
+    "eslint": "^8.23.1"
```

Comment:
Is this needed in the root package.json for some reason?  And if so, why not a devDependency?